lib/rewards/server/plugin.js

```diff
@@ -0,0 +1,38 @@
+/**
+ *
+ * Reldens - Rewards Server Plugin
+ *
+ */
+const {PluginInterface} = require("../../features/plugin-interface");
```

Comment:
cs: same as before, single quotes and space between the braces

Reply:
done

Reply:
`const {PluginInterface} = require("../../features/plugin-interface");``
>
`const { PluginInterface } = require('../../features/plugin-interface');`

not sure if the commit with the marked fixes was pushed?